dev-client/src/App.tsx

```diff
@@ -18,6 +18,8 @@ import {PermissionsAndroid} from 'react-native';
 import {Provider} from 'react-redux';
 import {createStore} from '../model/store';
 import {setAPIConfig} from 'terraso-client-shared/config';
+// needed for uuid - https://github.com/uuidjs/uuid#react-native--expo
+import 'react-native-get-random-values';
```

Comment:
suggestion: the docs state that the import should happen at the top of the import list